src/com/tony/server/response/PutPostResponse.java

```diff
@@ -1,11 +1,45 @@
 package com.tony.server.response;
 
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
 import java.util.HashMap;
 
 public class PutPostResponse extends Response {
-    public PutPostResponse() {
+    private final String filePath;
+    private String requestBody;
+
+    public PutPostResponse(String filePath) {
+        this.filePath = filePath;
         setStatusLine("HTTP/1.1 200 OK\n");
         setHeaders(new HashMap<>());
         setBody("");
     }
+
+    public byte[] respond(){
+        putPostContent();
+        return ResponseBuilder.buildResponse(getStatusLine(),
+                getHeaders(), getBody()).getBytes();
+    }
+
+    public void putPostContent() {
+        try {
+            PrintWriter writer = new PrintWriter(filePath, "UTF-8");
+            writer.print(requestBody);
+            writer.close();
```

Comment:
Can you think of a situation where this 'writer.close()' might not be executed? Search for 'try-with-resource' and 'finally' blocks in Java.